### ISSUE TITLE
crash report

### ISSUE THREAD

**Issue by Quasarship** [opentoonz/opentoonz#4959]
### Description

not sure whats wrong, but it asked me to fill out a report

### Steps to Reproduce

1. 
2. 

### Screenshots, Video & Crash Logs

OpenToonz 1.7.1  (Build Mar 14 2023)

Report Date: 2023-06-17 06:17:25
Crash Reason: EXCEPTION_ACCESS_VIOLATION

Build ABI: x86_64-little_endian-llp64
Operating System: Windows 10 Version 2009
OS Kernel: 10.0.19045
CPU Threads: 8

GPU Vendor: Intel
GPU Model: Intel(R) HD Graphics 630
GPU Version: 4.6.0 - Build 27.20.100.9664

Crash File: C:\Users\Owner\AppData\Local\OpenToonz\OpenToonz\crash\Crash-20230617-061725.log
Mini Dump File: C:\Users\Owner\AppData\Local\OpenToonz\OpenToonz\crash\Crash-20230617-061725.dmp

Application Dir: C:/Program Files/OpenToonz
Stuff Dir: C:\OpenToonz stuff

Project Name: face-turn
Scene Name: anticipation
Project Path: C:\OpenToonz stuff\projects\face-turn\face-turn_otprj.xml
Scene Path: C:\OpenToonz stuff\projects\face-turn\scenes\anticipation.tnz

==== Modules ====
C:\Program Files\OpenToonz\OpenToonz.exe
C:\Windows\SYSTEM32\ntdll.dll
C:\Windows\System32\KERNEL32.DLL
C:\Windows\System32\KERNELBASE.dll
C:\Windows\System32\USER32.dll
C:\Program Files\OpenToonz\Qt5Xml.dll
C:\Program Files\OpenToonz\Qt5PrintSupport.dll
C:\Windows\SYSTEM32\OPENGL32.dll
C:\Windows\System32\win32u.dll
C:\Windows\System32\msvcrt.dll
C:\Windows\System32\GDI32.dll
C:\Windows\System32\ADVAPI32.dll
C:\Windows\System32\gdi32full.dll
C:\Windows\System32\sechost.dll
C:\Windows\System32\ucrtbase.dll
C:\Windows\System32\COMDLG32.dll
C:\Windows\System32\msvcp_win.dll
C:\Windows\System32\RPCRT4.dll
C:\Windows\System32\combase.dll
C:\Program Files\OpenToonz\glut64.dll
C:\Windows\System32\SHELL32.dll
C:\Windows\System32\shcore.dll
C:\Windows\System32\ole32.dll
C:\Program Files\OpenToonz\Qt5SerialPort.dll
C:\Windows\System32\SHLWAPI.dll
C:\Program Files\OpenToonz\EDSDK.dll
C:\Windows\System32\OLEAUT32.dll
C:\Windows\System32\SETUPAPI.dll
C:\Windows\System32\cfgmgr32.dll
C:\Program Files\OpenToonz\image.dll
C:\Program Files\OpenToonz\colorfx.dll
C:\Windows\System32\WS2_32.dll
C:\Windows\System32\bcrypt.dll
C:\Program Files\OpenToonz\tnztools.dll
C:\Program Files\OpenToonz\tnzstdfx.dll
C:\Program Files\OpenToonz\tfarm.dll
C:\Program Files\OpenToonz\Qt5Widgets.dll
C:\Program Files\OpenToonz\Qt5Core.dll
C:\Program Files\OpenToonz\toonzqt.dll
C:\Program Files\OpenToonz\sound.dll
C:\Program Files\OpenToonz\Qt5Gui.dll
C:\Program Files\OpenToonz\opencv_world451.dll
C:\Program Files\OpenToonz\toonzlib.dll
C:\Program Files\OpenToonz\Qt5Script.dll
C:\Program Files\OpenToonz\tnzext.dll
C:\Program Files\OpenToonz\tnzbase.dll
C:\Program Files\OpenToonz\tnzcore.dll
C:\Program Files\OpenToonz\Qt5OpenGL.dll
C:\Program Files\OpenToonz\Qt5Multimedia.dll
C:\Program Files\OpenToonz\Qt5Network.dll
C:\Windows\SYSTEM32\MPR.dll
C:\Windows\SYSTEM32\GLU32.dll
C:\Windows\SYSTEM32\dbghelp.dll
C:\Windows\System32\CRYPT32.dll
C:\Program Files\OpenToonz\MSVCP140.dll
C:\Program Files\OpenToonz\MSVCP140_1.dll
C:\Program Files\OpenToonz\VCRUNTIME140.dll
C:\Windows\SYSTEM32\WINSPOOL.DRV
C:\Windows\SYSTEM32\WINMM.dll
C:\Windows\WinSxS\amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.19041.1110_none_792d1c772443f647\COMCTL32.dll
C:\Windows\SYSTEM32\MF.dll
C:\Program Files\OpenToonz\VCRUNTIME140_1.dll
C:\Windows\SYSTEM32\MFPlat.DLL
C:\Windows\SYSTEM32\VERSION.dll
C:\Windows\SYSTEM32\MSVFW32.dll
C:\Windows\SYSTEM32\AVIFIL32.dll
C:\Program Files\OpenToonz\libmypaint-1-4-0.dll
C:\Program Files\OpenToonz\glew32.dll
C:\Windows\SYSTEM32\UxTheme.dll
C:\Windows\SYSTEM32\dwmapi.dll
C:\Windows\SYSTEM32\USERENV.dll
C:\Windows\SYSTEM32\NETAPI32.dll
C:\Program Files\OpenToonz\Qt5Svg.dll
C:\Windows\SYSTEM32\dxgi.dll
C:\Windows\SYSTEM32\d3d11.dll
C:\Program Files\OpenToonz\CONCRT140.dll
C:\Windows\SYSTEM32\DNSAPI.dll
C:\Windows\SYSTEM32\IPHLPAPI.DLL
C:\Windows\SYSTEM32\MSACM32.dll
C:\Program Files\OpenToonz\libintl-8.dll
C:\Program Files\OpenToonz\libjson-c-2.dll
C:\Program Files\OpenToonz\libiconv-2.dll
C:\Windows\SYSTEM32\winmmbase.dll
C:\Windows\SYSTEM32\dbgcore.DLL
C:\Windows\SYSTEM32\NETUTILS.DLL
C:\Windows\SYSTEM32\MFCORE.DLL
C:\Windows\SYSTEM32\SRVCLI.DLL
C:\Windows\SYSTEM32\powrprof.dll
C:\Windows\SYSTEM32\ksuser.dll
C:\Windows\SYSTEM32\kernel.appcore.dll
C:\Windows\SYSTEM32\CRYPTBASE.DLL
C:\Windows\System32\IMM32.DLL
C:\Windows\System32\NSI.dll
C:\Windows\System32\bcryptPrimitives.dll
C:\Windows\SYSTEM32\RTWorkQ.DLL
C:\Windows\SYSTEM32\UMPDC.dll
C:\Program Files\OpenToonz\platforms\qwindows.dll
C:\Windows\SYSTEM32\WTSAPI32.dll
C:\Windows\SYSTEM32\windows.storage.dll
C:\Windows\SYSTEM32\Wldp.dll
C:\Windows\SYSTEM32\profapi.dll
C:\Program Files\OpenToonz\styles\qwindowsvistastyle.dll
C:\Program Files\OpenToonz\iconengines\qsvgicon.dll
C:\Windows\system32\d3d9.dll
C:\Windows\System32\MSCTF.dll
C:\Windows\SYSTEM32\textinputframework.dll
C:\Windows\System32\CoreUIComponents.dll
C:\Windows\System32\CoreMessaging.dll
C:\Windows\SYSTEM32\ntmarta.dll
C:\Windows\SYSTEM32\wintypes.dll
C:\Windows\System32\clbcatq.dll
C:\Windows\System32\AppXDeploymentClient.dll
C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_91e2144e79bce669\ig9icd64.dll
C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_91e2144e79bce669\igdml64.dll
C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_91e2144e79bce669\igdgmm64.dll
C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_91e2144e79bce669\igc64.dll
C:\Windows\system32\dataexchange.dll
C:\Windows\system32\dcomp.dll
C:\Windows\system32\twinapi.appcore.dll
C:\Windows\system32\dwrite.dll
C:\Program Files\OpenToonz\imageformats\qgif.dll
C:\Program Files\OpenToonz\imageformats\qicns.dll
C:\Program Files\OpenToonz\imageformats\qico.dll
C:\Program Files\OpenToonz\imageformats\qjpeg.dll
C:\Program Files\OpenToonz\imageformats\qsvg.dll
C:\Program Files\OpenToonz\imageformats\qtga.dll
C:\Program Files\OpenToonz\imageformats\qtiff.dll
C:\Program Files\OpenToonz\imageformats\qwbmp.dll
C:\Program Files\OpenToonz\imageformats\qwebp.dll
C:\Program Files\OpenToonz\bearer\qgenericbearer.dll
C:\Windows\SYSTEM32\dhcpcsvc6.DLL
C:\Windows\SYSTEM32\dhcpcsvc.DLL
C:\Windows\System32\netprofm.dll
C:\Windows\System32\npmproxy.dll
C:\Program Files\OpenToonz\audio\qtaudio_windows.dll
C:\Windows\SYSTEM32\MMDevAPI.DLL
C:\Windows\SYSTEM32\DEVOBJ.dll
C:\Windows\SYSTEM32\wdmaud.drv
C:\Windows\SYSTEM32\AVRT.dll
C:\Windows\SYSTEM32\AUDIOSES.DLL
C:\Windows\SYSTEM32\msacm32.drv
C:\Windows\SYSTEM32\midimap.dll
C:\Windows\SYSTEM32\resourcepolicyclient.dll
C:\Windows\System32\mswsock.dll
C:\Windows\System32\rasadhlp.dll
C:\Windows\system32\napinsp.dll
C:\Windows\system32\pnrpnsp.dll
C:\Windows\system32\wshbth.dll
C:\Windows\system32\NLAapi.dll
C:\Windows\System32\winrnr.dll
C:\Program Files\Bonjour\mdnsNSP.dll
C:\Windows\System32\fwpuclnt.dll
C:\Windows\SYSTEM32\LINKINFO.dll
C:\Windows\SYSTEM32\PROPSYS.dll
C:\Windows\system32\wintab32.dll
C:\Windows\system32\winhttp.dll
C:\Windows\SYSTEM32\SspiCli.dll
==== End ====

==== Backtrace ====
  1> RtlDeleteAce [0x7ffb169c8b9c] <ntdll.dll>
  2> RtlRaiseException [0x7ffb169a1316] <ntdll.dll>
  3> KiUserExceptionDispatcher [0x7ffb169f0f8e] <ntdll.dll>
  4> StyleEditorGUI::ColorSlider::setColor [0x7ffa7b54e237] <toonzqt.dll>
  5> StyleEditor::setStyle [0x7ffa7b54f1aa] <toonzqt.dll>
  6> StyleEditor::onStyleSwitched [0x7ffa7b54b8fe] <toonzqt.dll>
  7> QObject::qt_static_metacall [0x7ffa7b9c20d7] <Qt5Core.dll>
  8> TPaletteHandle::setPalette [0x7ffaa1be663b] <toonzlib.dll>
  9> PaletteController::editLevelPalette [0x7ffaa1c06630] <toonzlib.dll>
 10> TSmartPointerT<TXshPaletteLevel>::~TSmartPointerT<TXshPaletteLevel> [0x7ff6d5aefb70] <OpenToonz.exe>
 11> QObject::qt_static_metacall [0x7ffa7b9c20d7] <Qt5Core.dll>
 12> TSceneHandle::setScene [0x7ffaa1be6916] <toonzlib.dll>
 13> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff6d5cb2c28] <OpenToonz.exe>
 14> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff6d5c5eda0] <OpenToonz.exe>
 15> QObject::qt_static_metacall [0x7ffa7b9c20d7] <Qt5Core.dll>
 16> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff6d5e47dd0] <OpenToonz.exe>
 17> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff6d5c5d42a] <OpenToonz.exe>
 18> QWidget::event [0x7ffa9c18807c] <Qt5Widgets.dll>
 19> QFrame::event [0x7ffa9c229d10] <Qt5Widgets.dll>
 20> QApplicationPrivate::notify_helper [0x7ffa9c1655c0] <Qt5Widgets.dll>
 21> QApplication::notify [0x7ffa9c1634e0] <Qt5Widgets.dll>
 22> QCoreApplication::notifyInternal2 [0x7ffa7b9a318a] <Qt5Core.dll>
 23> QApplicationPrivate::sendMouseEvent [0x7ffa9c1667c5] <Qt5Widgets.dll>
 24> QDesktopWidget::workAreaResized [0x7ffa9c1b09d2] <Qt5Widgets.dll>
 25> QDesktopWidget::workAreaResized [0x7ffa9c1ae96e] <Qt5Widgets.dll>
 26> QApplicationPrivate::notify_helper [0x7ffa9c1655c0] <Qt5Widgets.dll>
 27> QApplication::notify [0x7ffa9c164643] <Qt5Widgets.dll>
 28> QCoreApplication::notifyInternal2 [0x7ffa7b9a318a] <Qt5Core.dll>
 29> QGuiApplicationPrivate::processMouseEvent [0x7ffa7bde47c3] <Qt5Gui.dll>
 30> QWindowSystemInterface::sendWindowSystemEvents [0x7ffa7bdcf030] <Qt5Gui.dll>
 31> QEventDispatcherWin32::processEvents [0x7ffa7b9ebcfa] <Qt5Core.dll>
 32> qt_plugin_query_metadata [0x7ffab6be28d9] <qwindows.dll>
 33> QEventLoop::exec [0x7ffa7b99f64c] <Qt5Core.dll>
 34> QCoreApplication::exec [0x7ffa7b9a2164] <Qt5Core.dll>
 35> TSmartPointerT<TXshSoundLevel>::~TSmartPointerT<TXshSoundLevel> [0x7ff6d5aa5219] <OpenToonz.exe>
 36> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff6d5eceab7] <OpenToonz.exe>
 37> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff6d5ece092] <OpenToonz.exe>
 38> BaseThreadInitThunk [0x7ffb152a7614] <KERNEL32.DLL>
 39> RtlUserThreadStart [0x7ffb169a26f1] <ntdll.dll>
==== End ====

### OpenToonz Version

1.7

### Operating System

Windows

### GPU

NVIDIA

### Graphics Tablet

None

**Comment by RodneyBaker** [comment]
Thanks for the report @Quasarship 
You may want to click through the crash report dialogue ignoring the prompts to close the program and see if that allows you to continue.  This has been working for some users.

Will close this report due to the following:
- Duplicate report to other recently filed crash reports
- Not enough information provided to assess the problem


Note to self (not necessarily related to your report here @Quasarship):  The bug report template likely needs to be updated as users are tending to accept defaults without supplying additional information.
For instance, the feedback may be skewed toward suggesting more users have Nvidia cards because "Nvidia" is the default (?).
That default might be better set to be a blank, "N/A" or some other entry that won't inadvertently skew the results.
In short, how can we improve the bug reporting template/process?

Note regarding the crash reporter:  What can be done to help avoid what apears to be false positives that lead to the crash report dialog activating?  There appears to be many cases where if the user navigates through the crash report dialogue... as if there wasn't a crash so choosing to press on with running Opentoonz... the program operates appropriately.
